app/helpers/audits_helper.rb

```diff
@@ -99,7 +99,7 @@ def audit_login?(audit)
   end
 
   def audit_action_name(audit)
-    return (audit.action == 'destroy') ? 'deleted' : "#{audit.action}d" if main_object? audit
+    return audit.action if main_object? audit
 
     case audit.action
```

Comment:
For main objects, you have removed the code which was added to form a past tense of the action.
To make it consistent, could you please modify the behavior for audits that are not belongs to main objects?

Reply:
updated!

Reply:
@jlsherrill, thank you for updates.
There are few places where proper action names handling needs to be done. 
With this PR changes, for add/remove actions it shows full attributes old/new changes but it is expected to show only the description that object is added or removed from main object. 
For example - 
![Screenshot from 2019-11-15 18-13-32](https://user-images.githubusercontent.com/6470528/68944374-b4bcfe00-07d3-11ea-9190-a01c6fa788a3.png)

1. In audit_helper at [line 85](https://github.com/theforeman/foreman/blob/develop/app/helpers/audits_helper.rb#L85) and [line 231](https://github.com/theforeman/foreman/blob/develop/app/helpers/audits_helper.rb#L231) 
2. Inside react component i.e AuditsList > ExpansiveView, at [line 51](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/AuditsList/ExpansiveView.js#L51)

Reply:
Thanks! Good catches!  I updated line 85 quite a bit to be i18n'd.